src/main/java/com/summitsync/api/model/Course.java

```diff
@@ -0,0 +1,35 @@
+package com.summitsync.api.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class Course {
+    @Id
+    @GeneratedValue
+    private long cId;
+    private int number;
+    @ManyToOne(cascade = CascadeType.ALL)
+    private Location location;
+    private String notes;
+    private boolean visible;
+    private double actualPrice; // in euros
```

Comment:
Für Geld sollte BigDecimal verwendet werden.
https://stackoverflow.com/questions/12496555/java-best-type-to-hold-price
Vielleicht sollten wir auch in postgres einen type benutzen, der auf zwei Nachkommastellen limitiert ist.
https://www.postgresql.org/docs/current/datatype-numeric.html
Wahrscheinlich NUMERIC oder DECIMAL. Keine Ahnung wie man das hier explizit angeben könnte.